scripts/release/rpm/mariner.dockerfile

```diff
@@ -15,13 +16,13 @@ COPY . .
 RUN dos2unix ./scripts/release/rpm/azure-cli.spec && \
     REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=python3 PYTHON_CMD=python3 \
     rpmbuild -v -bb --clean scripts/release/rpm/azure-cli.spec && \
-    cp /usr/src/mariner/RPMS/x86_64/azure-cli-${cli_version}-1.cm2.x86_64.rpm /azure-cli-dev.rpm
+    cp /usr/src/mariner/RPMS/*/azure-cli-${cli_version}-1.*.rpm /azure-cli-dev.rpm && \
+    mkdir /out && cp /usr/src/mariner/RPMS/*/azure-cli-${cli_version}-1.*.rpm /out/
```

Comment:
Any reason for copying to `/out`?

Reply:
In `scripts/release/rpm/pipeline.sh`,  we'll copy the rpm file with `docker cp $id:/out/. ${BUILD_STAGINGDIRECTORY}` later.
RPMs are in different folders on amd64 and arm64, but `docker cp` does not support wildcard. I move target package to `out` folder.